### PR TITLE
Name our routes, so /metrics gives more sensible aggregations

### PR DESCRIPTION
**Pull request by paulbellamy**
Previously was showing stuff like:
```
{route="api_pipe_pipe_8999861204239855606_probe"}   2
{route="api_pipe_pipe_2001043336351237495_check"}   1
{route="api_pipe_pipe_5613831799035630055"}
```

### CODE REVIEW

File: app/controls.go [hunk 1]
```diff
@@ -14,9 +14,14 @@ import (
 
 // RegisterControlRoutes registers the various control routes with a http mux.
 func RegisterControlRoutes(router *mux.Router, cr ControlRouter) {
-	router.Methods("GET").Path("/api/control/ws").
+	router.
+		Methods("GET").
+		Path("/api/control/ws").
 		HandlerFunc(requestContextDecorator(handleProbeWS(cr)))
```
Comment (jml) [inline app/controls.go]: Should this also have a `Name` call?

**Comment by jml** [comment]
One question, otherwise LGTM.

File: app/controls.go (hunk 1 above)
Comment (paulbellamy) [inline app/controls.go]: It has no dynamic segments to the path, so the instrumentation will infer it automatically.

Comment (jml) [inline app/controls.go]: Thanks.

**Comment by jml** [comment]
LGTM.

**Pull request merged**